mayastor-test/test_csi.js

```diff
@@ -80,6 +80,16 @@ function createPublishDir (mountTarget) {
   fs.mkdirSync(mountTarget);
 }
 
+function cleanBlockMount (blockfile, done) {
+  const proc = common.runAsRoot('umount', ['-f', blockfile]);
+  proc.once('close', (code, signal) => {
+    const proc = common.runAsRoot('rm', ['-f', blockfile]);
+    proc.once('close', (code, signal) => {
+      done();
+    });
+  });
```

Comment:
consider using execAsRoot for these that are simpler to use for one-time commands. runAsRoot is meant for long running processes that run in parallel to the test code (i.e. running mayastor).

Reply:
Done. https://github.com/openebs/Mayastor/pull/391/commits/72f4419b538bb6f8d967e948843a065087212eba